modules/@apostrophecms/ui/ui/apos/components/AposContextMenu.vue

```diff
@@ -9,15 +9,21 @@
       ref="dropdown"
       class="apos-popover__btn apos-context-menu__dropdown"
     >
+      <slot
+        v-if="slots.button"
+        name="button"
+        :on-click="buttonClicked"
+      />
```

Comment:
New feature in `AposContextMenu` a names slot `button` instead for traditional button. 
Used for rich text insert menu button that has a specific style.
Uses `on-click` prop to pass the method (events aren't supported here).